pkg/workceptor/lock_test.go

```diff
@@ -63,11 +63,11 @@ func TestStatusFileLock(t *testing.T) {
 				}
 				fileHasExisted = true
 				if err != nil {
-					t.Fatal(fmt.Sprintf("Error loading status file: %s", err))
+					t.Fatalf(fmt.Sprintf("Error loading status file: %s", err))
```

Comment:
Shouldn't all of those not just be `t.Fatalf("Error loading status file: %s", err)`?

Reply:
@eqrx you are right, will make the change now.